tools/i18n/unescape-contents

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import argparse
+import html
+import json
+import os
+import sys
+from contextlib import contextmanager
+from pathlib import PurePath
+from typing import IO, Any, Generator, NamedTuple, Union
+
+
+class CLIArgs(NamedTuple):
+    escape_html: bool
+    filename: str
+    indent_level: int
+
+
+def parse_args() -> CLIArgs:
+    parser = argparse.ArgumentParser(
+        prog="unescape-contents",
+        description="Unescape Unicode and, optionally, HTML entities in a JSON file. Input file must be a JSON dictionary. Output will always be unescaped UTF-8.",
+    )
+
+    parser.add_argument("filename", type=str)
+    parser.add_argument(
+        "--html",
+        action="store_true",
+        help="If the key of a dictionary field does not contain HTML escapes, unescape any HTML escapes found in the value",
+    )
+    parser.add_argument("--indent-level", dest="indent_level", type=int, default=2, required=False)
+    args = parser.parse_args()
+
+    return CLIArgs(args.html, args.filename, args.indent_level)
```

Comment:
I think we should call that first parameter `escape_html`, not `html`, so that you've got the same name used for this parameter consistently through the call chain; it'd be more readable that way.